# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/palemoon/hotfix-uuid.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Timebased uuid1() writes to /var/lib/libuuid/clock.txt, so we use random uuid4()

  - foxdrodd
--- Pale-Moon/platform/python/mach/mach/main.py.vanilla	2025-05-30 22:42:31.985000000 +0200
+++ Pale-Moon/platform/python/mach/mach/main.py	2025-05-30 22:42:41.594000000 +0200
@@ -260,7 +260,7 @@
                 mod = imp.new_module(b'mach.commands')
                 sys.modules[b'mach.commands'] = mod
 
-            module_name = 'mach.commands.%s' % uuid.uuid1().get_hex()
+            module_name = 'mach.commands.%s' % uuid.uuid4().get_hex()
 
         imp.load_source(module_name, path)
 
